packages/graphql/tests/tck/directives/cypher.test.ts

```diff
@@ -736,6 +736,7 @@ describe("Cypher directive", () => {
                     WITH this
                     CALL {
                         WITH this
+                        WITH this AS this
```

Comment:
This needs to be aliased? Can't just be `WITH this` again?

Reply:
In this particular case is not needed, but doesn't really seems to make sense to add extra logic in our codebase to treat this case any differently